docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -44,7 +44,10 @@ See [Troubleshooting](doc:troubleshooting#how-to-change-your-cache-directory) fo
 >    size_mb=$(du -m -d0 "${path}" | cut -f 1)
 >    if (( size_mb > limit_mb )); then
 >      echo "${path} is too large (${size_mb}mb), nuking it."
->      rm -rf "${path}"
+>      parent_dir=$(dirname "${path}")
```

Comment:
For an atomic mv we want the tmpdir to be on the same device, but not under the dir we're nuking, so a sibling seems safe.